mandatory/2-function-creation.js

```diff
@@ -45,7 +83,13 @@ function canRegister(age) {}
   )
 */
 
```

Comment:
Interesting implementation 😄 
Here you are calling the function from inside itself - this is called a recursive function.
It works! But for extra practice, can think of how you would do this with a loop instead? Maybe a `while` or `for` loop?